trade-mapping-to-ipaffs/src/main/java/uk/gov/defra/stw/mapping/toipaffs/chedpp/commodities/ChedppControlledAtmosphereContainerMapper.java

```diff
@@ -11,7 +11,7 @@ public class ChedppControlledAtmosphereContainerMapper implements
     Mapper<IncludedSpsTradeLineItem, ComplementParameterSetKeyDataPair> {
 
   private static final String CONTAINER_KEY = "container";
-  private static final String TRUE = "true";
+  private static final String TRUE = "TRUE";
```

Comment:
Changed because most of the values in UN/CEFACT are all caps, in terms of IPAFFS the value doesn't matter, when you set controlled atmosphere container in IPAFFS it actually sets this field to the species id number, but all it needs is a value that isn't null